tock/tock/templates/_navigation.html

```diff
@@ -10,11 +10,24 @@
           <span>Projects</span>
         </a>
       </li>
-      <li>
-        <a href="{% url 'reports:ListReports' %}" class="usa-nav-link">
-          <span>Reports</span>
-        </a>
-      </li>
+      {% if request.user.is_staff %}
+        <li>
+          <button class="
+            usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="side-nav-2">
```

Comment:
The `side-nav-2` is conflicting with the one under Administrative, this is causing the submenu not rendering.  So choose something else here.